packages/nc-gui-v2/composables/useViewColumns.ts

```diff
@@ -128,16 +121,16 @@ export function useViewColumns(view: Ref<ViewType> | undefined, meta: ComputedRe
     reloadData?.()
   }
 
-  const metaColumnById = computed(() => {
-    return (
-      meta.value?.columns?.reduce<Record<string, ColumnType>>((acc, curr) => {
-        return {
+  const metaColumnById = computed<Record<string, ColumnType>>(
+    () =>
+      meta.value?.columns?.reduce(
+        (acc: ColumnType, curr: ColumnType) => ({
           ...acc,
           [curr.id!]: curr,
-        }
-      }, {}) || {}
-    )
-  })
+        }),
+        {},
+      ) || {},
```

Comment:
Would probably be more readable to just do a quick check before.
Then you could also remove the `?` modifiers.

```ts
if (!meta.value?.columns) {
return {}
}
```